javascript/sources/entry-common.js

```diff
@@ -156,8 +156,8 @@ module.exports = function (bind, lib) {
 
             }
 
-            if (!Object.prototype.hasOwnProperty.call(methods, unit))
-                throw new Error(`Failed to execute "${fnName}": Unsupported unit.`);
+            if (!methods[unit])
```

Comment:
@larsonjj can you detail what value was causing the previous check to fail? I don't see what this fix 😃

Reply:
Hey @arcanis,

I was getting incorrect results when trying to pass `auto` value into `setMaxWidth` function. It seems that `auto` is not a supported value, but in the original code, `(!Object.prototype.hasOwnProperty.call(methods, unit))` was returning `false` when it should return `true`.

Reply:
Aaaaah yes I see, thanks! :D